experimental/IntersectionTheory/src/Main.jl

```diff
@@ -50,6 +50,22 @@ true
 abstract_bundle(X::AbstractVariety, ch::MPolyDecRingOrQuoElem) = AbstractBundle(X, ch)
 abstract_bundle(X::AbstractVariety, r::RingElement, c::MPolyDecRingOrQuoElem) = AbstractBundle(X, r, c)
 
+#######################################################
+@doc raw"""
+    ==(F::AbstractBundle, G::AbstractBundle)
+
+Return `true` if `F` is equal to `G`, and `false` otherwise.
+
+# Examples
+```jldoctest
+julia> P2 = abstract_projective_space(2)
+AbstractVariety of dim 3
+
+julia> 3*OO(P2, 1) - OO(P2) == tangent_bundle(P2) # Euler sequence
+true
+
+```
+"""
 ==(F::AbstractBundle, G::AbstractBundle) = chern_character(F) == chern_character(G)
```

Comment:
every type with a custom `==` function should have a `hash` function as well (according to julia documentation and julia base assumptions). See https://github.com/oscar-system/Oscar.jl/issues/2222 for details. For this particular case, you can just delegate the hashing to the chern character as that already determines the `AbstractBundle`.
```suggestion
==(F::AbstractBundle, G::AbstractBundle) = chern_character(F) == chern_character(G)

function Base.hash(F::AbstractBundle, h::UInt)
  return hash(chern_character(F), h)
end
```

Reply:
done with [7b085d7](https://github.com/oscar-system/Oscar.jl/pull/4269/commits/7b085d7e30a4975ae4091a6765808bcf20bbf988)

Reply:
Thanks!